admin-ui/src/pages/settings-page.tsx

```
@@ -212,6 +269,73 @@ function parseReasoningJson(
   }
 }
 
+function parseModelAliasesJson(
+  value: string,
+): ParseResult<ModelAliasRecord> {
```

ESLint

[error] Replace `⏎··value:·string,⏎` with `value:·string`

(prettier/prettier)